lldb/examples/python/crashlog.py

```diff
@@ -1720,6 +1735,13 @@ def CreateSymbolicateCrashLogOptions(
         help="show source for all threads, not just the crashed thread",
         default=False,
     )
+    arg_parser.add_argument(
+        "--no-parallel-image-loading",
+        dest="parallel_image_loading",
```

Comment:
The name of the flag and the variable seem to be opposites. Does argparse automatically change the value because the flag contains `-no`.